t/python/t0022-resource-set.py

```diff
@@ -41,6 +41,27 @@ class TestRSet(unittest.TestCase):
       }
     }
```

Comment:
commit message typo "secondard" -> "second"? "secondary"?

Reply:
Thanks! Fixed and will set MWP.